client/src/routes.js

```diff
@@ -19,13 +20,34 @@ export default (
       getComponent={(location, callback) => callback(null, Home)}
     />
     <Route
+      path="/choose-campaign"
+      getComponent={(location, callback) => callback(null, ChooseCampaign)}
+    />
+    <Route
       path="/campaign"
-      onEnter={(nextState, replace) => !nextState.params.id && replace('/campaign/new')}
+      onEnter={(nextState, replace) => !nextState.params.id && replace('/new-campaign')}
     />
     <Route
-      path="/campaign/new"
+      path="/new-campaign"
       getComponent={(location, callback) => callback(null, NewCampaign)}
-    />
+    >
+      <Route
+        path="/new-campaign/address"
+        getComponent={(location, callback) => callback(null, NewCampaign)}
```

Comment:
These four routes will obviously need to call a child component of NewCampaign, once we build those out.